tests/asserts/final/publication/internal.py

```diff
@@ -188,7 +188,7 @@ def correct_values_in_detail(workspace, publ_type, name, *, exp_publication_deta
                                        {
                                            '_file': {
                                                'paths': {
-                                                   os.path.splitext(os.path.basename(filename))[0] if len(filenames) > 1 else name:
+                                                   input_file.slugify_timeseries_filename(os.path.splitext(os.path.basename(filename))[0]) if len(filenames) > 1 else name:
```

Comment:
I do not like using input_file.slugify_timeseries_filename 4 times, when I do not know the reason.